packages/main/src/plugin/container-registry.ts

```diff
@@ -1071,6 +1071,70 @@ export class ContainerProviderRegistry {
     }
   }
 
+  // Start containers that match a label + key
+  async startContainersByLabel(engineId: string, label: string, key: string): Promise<void> {
+    let telemetryOptions = {};
+    try {
+      // Get all the containers using listSimpleContainers
+      const containers = await this.listSimpleContainers();
+
+      // Find all the containers that are using projectLabel and match the projectName
+      const containersMatchingProject = containers.filter(container => {
+        // dont try to start it if it's already running
+        if (container.State === 'running') {
+          return false;
+        }
+        const labels = container.Labels;
+        return labels && labels[label] === key;
+      });
```

Comment:
I have not tested but could be a one-liner
```suggestion
      // Find all the containers that are not already running and using projectLabel and match the projectName
      const containersMatchingProject = containers.filter(container => container.State !== 'running' && container.Labels?.[label] === key);
```